stdlib/rlp.mini

```diff
@@ -38,6 +39,17 @@ import func hasher_pushN_le(h: Hasher, val: uint, nbytes: uint) -> Hasher;
 import func hasher_pushN_be(h: Hasher, val: uint, nbytes: uint) -> Hasher;
 import func hasher_finish(h: Hasher) -> bytes32;
 
+type SignedTx = struct {
```

Comment:
This data structure is duplicated between here and signedtx.mini. There should be a comment on both of them documenting that

Reply:
Done.